src/pages/command-menu.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Command Menu
+pageTitle: Command Menu | Plane
+description: Simple and easy to understand the Command Menu.
```

Comment:
Learn how to quickly navigate through Plane with your keyboard using the Command Menu.